tests/functional/BaseCase.php

```diff
@@ -336,4 +333,51 @@ protected function assertEqualsArrays(array $expected, array $actual): void
 
         $this->assertEquals($expected, $actual);
     }
+
+    /**
+     * @return array{CreateBucketResponse, Connection}
+     */
+    protected function createTestBucket(
+        GenericBackendCredentials $projectCredentials,
+        CreateProjectResponse $projectResponse
+    ): array {
+        $bucket = md5($this->getName()) . '_Test_bucket';
+
+        $handler = new CreateBucketHandler($this->sessionManager);
+        $command = (new CreateBucketCommand())
+            ->setStackPrefix($this->getStackPrefix())
+            ->setProjectId($this->getProjectId())
+            ->setBucketId($bucket)
+            ->setProjectRoleName($projectResponse->getProjectRoleName())
+            ->setProjectReadOnlyRoleName($projectResponse->getProjectReadOnlyRoleName());
+
+        $response = $handler(
+            $projectCredentials,
+            $command,
+            []
+        );
+        $this->assertInstanceOf(CreateBucketResponse::class, $response);
+
+        $db = $this->getConnection($projectCredentials);
+
+        $this->assertTrue($this->isDatabaseExists($db, $response->getCreateBucketObjectName()));
+
+        // read only role has read access to bucket
+        $this->assertEqualsArrays(
+            ['R '],
```

Comment:
po novu je na to konstanta v TeradataAccessRight

Reply:
done